docs/test.ts

```diff
@@ -124,8 +124,16 @@ const elasticsearchOptions = {
   },
 };
 
+const awsSdkOptions = {
+  service: 'test',
+  hooks: {
+    addCustomTags: (span, params) => {},
```

Comment:
Ok understood, that makes more sense to me now. I've updated this and also (regarding later comment) am just passing in the full AWS.Response Object. This will give users access to the response, response.request and response.request.params so they should be able to add any relevant metadata tags via this hook.